tests/karate/src/test/java/fe/utils/verification/ElectionVerification.java

```diff
@@ -0,0 +1,92 @@
+package fe.utils.verification;
+
+import be.utils.JsonConverter;
+import com.intuit.karate.Json;
+import com.intuit.karate.Logger;
+
+import java.security.NoSuchAlgorithmException;
+import java.util.List;
+
+import static common.utils.Constants.*;
+import static fe.utils.verification.VerificationUtils.getMsgDataJson;
+import static fe.utils.verification.VerificationUtils.getStringFromIntegerField;
+
+/** This class contains functions used to test fields specific to Roll-Call */
+public class ElectionVerification {
+  private static final Logger logger = new Logger(ElectionVerification.class.getSimpleName());
+
+  /**
+   * Verfies that the election id is coherently computed
+   * @param message the network message
+   * @return true if the computed election id matches what is expected
+   */
+  public boolean verifyElectionId(String message) {
+    Json setupMessageJson = getMsgDataJson(message);
+
+    String electionId = setupMessageJson.get(ID);
+    String createdAt = getStringFromIntegerField(setupMessageJson, CREATED_AT);
+    String laoId = setupMessageJson.get(LAO);
+    String electionName = setupMessageJson.get(NAME);
+
+    try {
+      JsonConverter jsonConverter = new JsonConverter();
```

Comment:
This definitely needs to be renamed 😅

Reply:
I could not agree more. I won't do it here though